### PR TITLE
mk_oracle: Introduce Logging Improvements

### PR DESCRIPTION
**Pull request by Bastian-Kuhn**
This request introduces serval improvements for the mk_oracle logging.

### CODE REVIEW

**Comment by jherbel** [comment]
Hi Bastian,
thank you for this PR, it is queued for internal review. Please not that this might take a while, since we are currently very busy with the release of checkmk 1.7.
Best
Jörg
Internal reference: CMK-5436

**Comment by Rendanic** [comment]
This enhancement looks really nice. It should be added to the master.

File: agents/plugins/mk_oracle [hunk 1]
```diff
@@ -2171,7 +2172,7 @@ do_custom_sqls () {
                 # local connection mode
                 local SQLS_ITEM_SID="$MK_SID"
             fi
-            logging "[${MK_SID}] [${section}] [custom_sql] "\
+            logging "[${MK_SID}] [custom_sql] [${section}] "\
```
Comment (si-23) [inline agents/plugins/mk_oracle:2175]: Trailing space.. but it's ok for now

**Review by si-23 — APPROVED** [review]

**Comment by si-23** [comment]
Will be merged

**Pull request closed without merging**